.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+name: Integrity check
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    
+    - name: Install PHP
+      uses: shivammathur/setup-php@master
+      with:
+        php-version: 7.4
```

Comment:
`shivammathur/setup-php` is a pretty cool, pretty useful GitHub Action IMO. Just thought I'd mention though, since I notice this PR here seems to utilise the action only to test against PHP 7.4; `shivammathur/setup-php` makes it possible to test against multiple PHP versions.

E.g., instead of:
```YAML
jobs:
  build:
    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@master

    - name: Install PHP
      uses: shivammathur/setup-php@master
      with:
        php-version: 7.4
```

You could choose to do something like:
```YAML
jobs:
 build:
  runs-on: ${{ matrix.operating-system }}

  strategy:
   matrix:
    operating-system: [ubuntu-latest]
    php-versions: ['5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0']
  name: PHP ${{ matrix.php-versions }} on ${{ matrix.operating-system }}
  steps:
  - name: Checkout
    uses: actions/checkout@v2

  - name: Set up PHP
    uses: shivammathur/setup-php@v2
    with:
     php-version: ${{ matrix.php-versions }}
```

..thus, allowing to test against all PHP versions currently supported by the package (just as an example). :-)